ui_framework/components/icon/delete_icon.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { KuiIcon } from './kui_icon';
+
+export function DeleteIcon() {
+  return <KuiIcon className="fa-trash"/>;
```

Comment:
> As these become a whole lot of tiny files, I think I'd personally go for starting with adding them to ui_framework/components/icon/index.js like this:

Personally I don't mind separate files (as you can guess... I like consistency).

Also, I wonder if we should use CamelCasing for the component file names (will clearly indicate that a file contains a component). In this example, `DeleteIcon.js`, `CreateIcon.js`, `KuiIcon.js`, etc...

> then later on extract even further if needed. That would also make it easy to extract one icon into it's own file and do something like this:

How about just always using named imports?